### ISSUE TITLE
Remove console logs

### ISSUE THREAD

**Issue by prasanna-5796** [deepakgonda/angular-image-viewer#4]
Can you update the code and remove console logs, which exposes image url and poses security risk

**Comment by deepakgonda** [comment]
Yes, I will remove it in next version, but I want to mention it is not a security risk, b/c if we remove it from console log then also you can view the urls of all the images in the networks tab.

**Comment by ghost** [comment]
I'm hoping we console logs will be removed soon on the next release - thanks a lot

**Comment by koralLevi** [comment]
Can you please update the package and remove the console logs that prints the base64 from the code?
thanks
